elements/rh-tag/docs/20-guidelines.md

```diff
@@ -0,0 +1,155 @@
+{% section %}
+## Usage
+
+Use a tag to highlight an element on a page to draw attention to it or make it 
+more searchable.
+
+## Tag vs. Badge
+
+If you need to reflect counts like number of objects, events, or unread items, 
+use a Badge instead.
+
+## Variations
+Both variations come in `red`, `orange`, `green`, `cyan`, 
+`blue`, `purple`, and `gray` colors. The white 
+variation is for the dark theme only. A filled tag can be used to add more 
+visual prominence whereas an unfilled tag can be used for grouping. You can use 
+both variations in the same layout or user interface, just not in the same area 
+or container. Whatever you choose, be sure to maintain consistency as best as 
+possible.
+
+{% alert state='warning', title='Warning' %}
+Relying on color alone to communicate information causes barriers to access for 
+many users. Learn more in the [Accessibility section](/accessibility).
+{% endalert %}
+
+{% example palette="light",
+        width=404,
+        alt="A row of filled tags with text ‘Filled tags’ underneath and a row of unfilled tags with text ‘Unfilled tags’ underneath",
+        src="../tag-variations-colors.png" %}
+
+## Icons
+
+Add an icon when additional visual information is helpful or to distinguish tags 
+of the same color.
+
+{% alert title="Helpful tip" %}
+Add an icon when additional visual information is helpful or to distinguish 
+tags of the same color.
+{% endalert %}
+
+{% example palette="light",
+        width=475,
+        alt="A row of tags showing examples of optional icons",
+        src="../tag-icons.png" %}
+
+## White Tag
+
+The unfilled white tag should be used in the dark theme or on dark backgrounds. 
+It should also be used on its own and not be grouped. The text should not 
+indicate a status, it should be written to be a descriptive caption to elements 
+nearby.
+
+{% example palette="light",
+        width=1000,
```

Comment:
this overflows the page

![image](https://github.com/RedHat-UX/red-hat-design-system/assets/1466420/56390307-7063-45e6-99ff-ce5b7ee29f70)

Reply:
Looks like this is a problem on many images depending on the sizing of the screen.  [Added some css here](https://github.com/RedHat-UX/red-hat-design-system/blob/be2cb48f41251224995810f2f4df03e14d92dccd/docs/scss/components/_example.scss#L215) to fix this for the images.  Though they end up looking pretty small on smaller screens without zooming in.